core/sawtooth/cli/monitor.py

```diff
@@ -1,261 +0,0 @@
-# Copyright 2016 Intel Corporation
```

Comment:
It breaks 'git blame' and all such history (to remove then copy back in and commit as a new author from scratch), which is why it didn't get purged previously.

I wonder how much effort it would be to simplify the stats code to what's currently supported (which doesn't include most of the metrics) only.  The fork detect code, for example, should still work with what we currently get out of the REST API since it's using block id lists.